bindings/c/test/mako/stats.hpp

```diff
@@ -0,0 +1,157 @@
+/*
+ * stats.hpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MAKO_STATS_HPP
+#define MAKO_STATS_HPP
+
+#include <array>
+#include <cstdint>
+#include <cstring>
+#include <list>
+#include <utility>
+#include "operations.hpp"
+#include "time.hpp"
+
+namespace mako {
+
+/* size of each block to get detailed latency for each operation */
+constexpr const size_t LAT_BLOCK_SIZE = 4095;
+
+/* memory block allocated to each operation when collecting detailed latency */
+class LatencySampleBlock {
+	uint64_t samples[LAT_BLOCK_SIZE]{
+		0,
+	};
+	uint32_t index{ 0 };
```

Comment:
This should be `uint64_t` to make LatencySampleBlock exactly 8 * 4096 = 32KB.

Reply:
Since std::list is based on doubly-linked list, it adds 16B to allocation unit. So I'll make LAT_BLOCK_SIZE = 4093. (trailing 4B gets padded anyway, but I'll use uint64_t for index just to fill it up).